public/javascripts/ViewCarts.js

```diff
@@ -47,12 +47,14 @@ function displayCartInventory() {
     var idSelected = $("#selectDropDown :selected").val();
     $("#qr_button").removeClass("hidden");
 
-    navigation.hit("/Carts/GetCartItems/" + encodeURIComponent(idSelected), function (data) {
-        data = JSON.parse(data);
-        populateCartContainer(data[0]);
-    })/*.fail(function(res) {
-        $("#response").text("Error: displayCartInventory: Connection error.");
-    })*/;
+    navigation.getJSON(window.apiRoute + "/Carts/GetCartItems/" + encodeURIComponent(idSelected), function (err, data) {
```

Comment:
The getJSON just does the parsing for you using the jQuery Ajax code. It makes our code simpler to use this method if you are expecting json back and manually parsing it.